toggles/webapp/toggles.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
   toggles: [
+    {
+      id: 'selectableQueries',
+      title: `Selectable queries`,
+      defaultValue: false,
+      description:
+        'allow people to use different query types while seaching the catalogue',
```

Comment:
typo: searching

Reply:
Maybe uppercase 'Allow' for consistency.

Reply:
And a full-stop, because I'm a stickler for all the grammars.

Reply:
Done